tests/unit/test_signature.cpp

```diff
@@ -771,35 +771,39 @@ TEST_F(SignatureTest, testUnsuccessfulEccVerificationWithWrongPublicKey)
  */
 TEST_F(SignatureTest, testSuccessfulEccVerificationInIEEE1363Format)
```

Comment:
Should we also add a test that verifies that the default serialization format is the ASN.1 sequence of integers by creating a signature with this encoding and attempting to verify it with the default encoding?

Reply:
This is what
`TEST_P(SignatureTest, testSuccessfulSignatureAndVerification)`
implicitly already does. We also check that the default verification format is the ASN.1 one by the tests that check against the known good signatures (which are ASN.1 encoded).